app/src/main/java/dominando/android/livros/BookDetailsFragment.kt

```diff
@@ -8,51 +8,38 @@ import android.view.MenuItem
 import android.view.View
 import android.view.ViewGroup
 import androidx.databinding.DataBindingUtil
-import androidx.lifecycle.Observer
 import dominando.android.livros.common.BaseFragment
 import dominando.android.livros.databinding.FragmentBookDetailsBinding
 import dominando.android.presentation.BookDetailsViewModel
-import dominando.android.presentation.ViewState
 import dominando.android.presentation.binding.Book
 import org.koin.android.ext.android.inject
+import org.koin.core.parameter.parametersOf
 
 class BookDetailsFragment : BaseFragment() {
-    private val viewModel: BookDetailsViewModel by inject()
 
-    private lateinit var binding: FragmentBookDetailsBinding
+    private val viewModel: BookDetailsViewModel by inject {
+        parametersOf(arguments?.getParcelable<Book>("book")?.id)
```

Comment:
I know it's from existing code, but could you create a constant for the argument key? e.g.: `EXTRA_BOOK`

Reply:
I created a Constants class to put all public constants